build/build.sh

```diff
@@ -1,12 +1,18 @@
-set -o errexit
+#!/usr/bin/env bash
+
+## build.sh: compile manuscript outputs from content using Manubot and Pandoc
+
+set -o errexit \
+    -o nounset \
+    -o xtrace
 
 # Set timezone used by Python for setting the manuscript's date
 export TZ=Etc/UTC
 # Default Python to read/write text files using UTF-8 encoding
 export LC_ALL=en_US.UTF-8
 
 # Generate reference information
-echo "Retrieving and processing reference metadata"
+>&2 echo "Retrieving and processing reference metadata"
```

Comment:
I'm not accustomed to seeing `>&2` at the beginning of the line, but it looks like there is not single convention for this.

Reply:
I agree this seems rather odd to me but I did not comment on it earlier. More important is a consistent use of whatever convention you choose. My own preference in descending order would be:

```
echo >&2 "Retrieving and processing reference metadata"
echo "Retrieving and processing reference metadata" >&2
>&2 echo "Retrieving and processing reference metadata"
```

Normally I do redirections at the end of a command but with `echo` or `printf` it can get lost at the end.